.github/workflows/release.yaml

```diff
@@ -0,0 +1,22 @@
+name: Release Karavi-Resiliency
+# Invocable as a reusable workflow
+# Can be manually triggered
+on: # yamllint disable-line rule:truthy
```

Comment:
yaml lint warning

Reply:
Hi @shaynafinocchiaro This is a false positive and hence you only enabled skipping this truth check lint issue. Here is the PR https://github.com/dell/csi-powerstore/pull/376/files#diff-742d5fb583c9ace35d1e6110969c7601592c2d7b97901a9fc6d5e5a59bbd320fR2 where you have introduced this and @gallacher has merged it.

Reply:
It's looking for an additional space between `on:` and `#`